spec/unit/application/knife_spec.rb

```diff
@@ -75,11 +77,7 @@ def run
       expect(@knife).to receive(:exit).with(0)
       @knife.run
     end
-    if windows?
```

Comment:
that's not a weird if statement at all

Reply:
Yeah - just found this randomly, figured I would do some cleanup.